### PR TITLE
Update README

### LINKED ISSUE #18: Improve README
The README is out of date and incomplete. It's really the manual for this utility, and it's inadequate. Some things to fix:

- [x] Update the documentation around JQ_PY_VENV
- [x] Add all the sub-commands
- [x] Document the commands
- [x] Explain why the rules around state exist

### PR DESCRIPTION
**Pull request by rod-glover**
Resolves #18

### CODE REVIEW

**Comment by rod-glover** [comment]
@jameshiebert if you care to -- and only if you care to -- you could have a look at these changes from the perspective of a user and let me know if there's anything else/different you'd suggest.

File: README.rst [hunk 1]
```diff
@@ -68,22 +68,34 @@ There's little to configure in JQ, just a couple of environment variables and th
...
+  JQ from!
+  You *can* run JQ from the predefined environments below, because
+  for convenience they have JQ installed as well as ``generate_climos``.)
+
+There are a predefined environments suitable for JQ jobs at
+
+- ``/storage/data/projects/comp_support/climate_exporer_data_prep/climatological_means/venv``
+- ``/storage/data/projects/comp_support/climate_exporer_data_prep/climatological_means/venv_climdex``
+
+which should be used in most cases. xxx
```
Comment (jameshiebert) [inline README.rst]: "xxx"?

File: README.rst [hunk 2]
```diff
@@ -68,22 +68,34 @@ There's little to configure in JQ, just a couple of environment variables and th
...
 
 In short, your JQ configuration should normally be::
 
-    $ export JQ_DATABASE=/storage/data/projects/comp_support/climate_exporer_data_prep/climatological_means/jobqueue-prod.sqlite
-    $ export JQ_PY_VENV=/storage/data/projects/comp_support/climate_exporer_data_prep/climatological_means/venv
+    $ export JQ_DATABASE=/storage/data/projects/comp_support/climate_explorer_data_prep/climatological_means/jobqueue-prod.sqlite
```
Comment (jameshiebert) [inline README.rst:97]: Ha :)

File: README.rst [hunk 3]
```diff
@@ -105,75 +117,166 @@ Actions overview
...
+
+``script``          Write to stdout the script that will be or was submitted by ``jq submit`` for
+                    a specified queue entry.
+
+``submit``          Dequeue one or more ``generate_climos`` queue entries with NEW status,
+                    and submit a PBS job for each, updating the queue entries accordingly.
+
+``summarize``       Summarize entries in ``generate_climos`` queue (generate statistics)
+
+``update-email``    Update ``generate_climos`` queue using PBS status email
```
Comment (jameshiebert) [inline README.rst]: update-email baffled me as a user. Should we just remove this option? update-qstat works great.

**Review by jameshiebert — COMMENTED** [review]
Looks pretty good to me.

File: README.rst (hunk 3 above)
Comment (rod-glover) [reply README.rst]: It's useful if you waited too long to do an update-qstat. Eventually PBS dumps info and you can't get it back. But I agree it's not much use most of the time, plus it's laborious to use. Punishment for the tardy.

**Pull request merged**